m3u8/model.py

```diff
@@ -263,6 +263,13 @@ def dumps(self):
             output.append(str(self.start))
         if self.is_i_frames_only:
             output.append('#EXT-X-I-FRAMES-ONLY')
+        if self.segment_map:
+            map_output = []
+            for attr, value in self.segment_map.items():
+                if '"' not in value:
```

Comment:
I needed to do it because the `URI` comes without the quotes. They are even defined in the RFC:
https://tools.ietf.org/html/rfc8216#section-4.3.2.5

```
Its format is:

   #EXT-X-MAP:<attribute-list>

   The following attributes are defined:

      URI

      The value is a quoted-string containing a URI that identifies a
      resource that contains the Media Initialization Section.  This
      attribute is REQUIRED.
````

There is a part in the code that removes it. I don't know the reasons behind this but I am pretty sure they exist.